docs/extensions.rst

```diff
@@ -47,10 +47,12 @@ additional methods:
 .. method:: jinja2.Environment.install_gettext_translations(translations, newstyle=False)
```

Comment:
i think the signature here needs to be updated (or do we even need the signature? why can't babel get it from the actual method?)

Reply:
@davidism any idea why we have explicit signatures in the docs?

Reply:
I didn't write them, it's one of the many inconsistencies throughout the docs that I clean up when I notice. Should be safe to let Sphinx autodoc get it. Right now it's not using autodoc, so the full signature needs to be written out manually.

Reply:
@ThiefMaster I updated `versionchanged`. But don't know what to do with this

Reply:
for now I'd just update the method's signature and add the new arguments

Reply:
Sorry, maybe because I am not as familiar with sphinx I don't follow what you mean. I've already added information about `pgettext` and `npgettext` in docs description for `install_gettext_translations`. Where are as well this signatures should be updated?

Reply:
ah sorry, i just realized that we're just passing `translations` here and not the individual functions, so nevermind